frontend/src/components/shortener-input/shortener-input.tsx

```diff
@@ -10,13 +10,21 @@ export interface ShortenerInputProps {
 
 export const ShortenerInput = component$((props: ShortenerInputProps) => {
   const state: Store = useContext(InputContext) as Store;
+  const inputRef = useSignal<HTMLInputElement>();
 
   return (
     <div className="form-control">
       <div class="sm:input-group mb-3 flex-col sm:flex-row gap-2 sm:gap-0">
         <input
           onKeyUp$={props.onKeyUp$}
           onInput$={props.onInput$}
+          ref={inputRef}
+          document:onKeyDown$={(e: KeyboardEvent) => {
+            if (e.key === '/') {
+              e.preventDefault();
+              inputRef.value?.focus();
+            }
+          }}
```

Comment:
Hey @xarielah,

I think it will be much cleaner if you will separate the keyDown handler to a function.

```js
const onKeyDown = (e: KeyboardEvent) => {
  if (e.key === '/') {
    e.preventDefault();
    inputRef.value?.focus();
  }
};
```
And change the onKeyDown event handler:
```js
onKeyDown={onKeyDown}
```

Reply:
So sure I will apply the change today, and will update as soon as it is done 🚀